### PR TITLE
Improve code organization consistency

### PR DESCRIPTION
**Pull request by vasilmkd**
I hope you like it.

### CODE REVIEW

File: keeper/src/main/scala/zio/keeper/membership/swim/SWIM.scala [hunk 1]
```diff
@@ -24,7 +25,7 @@ import scala.collection.immutable.SortedSet
 
 object SWIM {
 
-  def join(
+  def live(
```
Comment (mijicd) [inline keeper/src/main/scala/zio/keeper/membership/swim/SWIM.scala]: I would even add aliases to these environments, as they are getting a bit out of hand.

File: keeper/src/main/scala/zio/keeper/discovery/static/Static.scala [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package zio.keeper.discovery.static
+
+import zio.{ Layer, UIO, ZLayer }
+import zio.keeper.discovery.Discovery
+import zio.nio.core.SocketAddress
+
+object Static {
```
Comment (mijicd) [inline keeper/src/main/scala/zio/keeper/discovery/static/Static.scala]: Maybe just leave it as package object?

File: keeper/src/main/scala/zio/keeper/transport/tcp/Tcp.scala [hunk 3]
```diff
@@ -7,12 +7,13 @@ import zio.clock.Clock
...
 import zio.keeper.TransportError
 import zio.keeper.TransportError._
+import zio.keeper.transport._
 import zio.keeper.transport.Channel._
 import zio.logging._
 import zio.nio.channels._
 import zio.nio.core.SocketAddress
 
-object tcp {
+object Tcp {
```
Comment (mijicd) [inline keeper/src/main/scala/zio/keeper/transport/tcp/Tcp.scala]: I'd rather keep the old encoding here, but it's a personal preference.

**Comment by vasilmkd** [comment]
I'll do another pass. Thanks for your thoughts @mijicd.

**Comment by pshemass** [comment]
@vasilmkd reopen this when you rebase it with latest master.

**Pull request closed without merging**